test/Properties.hs

```diff
@@ -44,10 +45,19 @@ t_normalize mode = t_nonEmpty $ T.normalize mode
 
 main :: IO ()
 main = do
+    putStrLn "quotRem28"
+    check $ \n -> n `quotRem` 28 == quotRem28 n
+    putStrLn "quotRem21"
+    check $ \n -> n `quotRem` 21 == quotRem21 n
```

Comment:
* Strictly speaking, `quotRem21` and `quotRem28` are valid only for non-negative inputs, I'm a bit surprised that they pass as is.
* It is important to check the upper end of range as well, like `check $ \n -> quotRem (maxBound - n) 21 == quotRem 21 (maxBound - n)`.

Reply:
The tests were indeed failing, I fixed that.

The division works correctly upto `maxBound / 2`, fails beyond that.